v3/src/models/app-state.ts

```diff
@@ -0,0 +1,94 @@
+/*
+  AppState
+
+  AppState is for application state that is not intended for serialization.
+  It is currently used to support an `appMode` property which can be used to alter behavior
+  in performance-critical contexts, e.g. during a drag. The properties of this class will
+  generally be MobX-observable.
+ */
+import { action, computed, makeObservable, observable } from "mobx"
+import { getEnv } from "mobx-state-tree"
+import { createCodapDocument } from "./codap/create-codap-document"
+import { gDataBroker } from "./data/data-broker"
+import { IDocumentModel, IDocumentModelSnapshot } from "./document/document"
+import { ISharedDataSet, kSharedDataSetType } from "./shared/shared-data-set"
+import { ITileEnvironment } from "./tiles/tile-content"
+
+type AppMode = "normal" | "performance"
+
+class AppState {
```

Comment:
Note that this file is not new -- it just changed location and had the document added to it.

Reply:
Got it